src/NodeJS/NodeJSServiceImplementations/OutOfProcess/NodeJSProcessFactory.cs

```diff
@@ -48,7 +48,7 @@ internal ProcessStartInfo CreateStartInfo(string nodeServerScript)
 #endif
             var startInfo = new ProcessStartInfo(_nodeJSProcessOptions.ExecutablePath!) // ConfigureNodeJSProcessOptions sets ExecutablePath to "node" if user specified value is null, whitespace or an empty string
             {
-                Arguments = $"{_nodeJSProcessOptions.NodeAndV8Options} -e \"{nodeServerScript}\" -- --parentPid {currentProcessPid} --port {_nodeJSProcessOptions.Port}",
+                Arguments = $"{_nodeJSProcessOptions.NodeAndV8Options} --input-type=module -e \"{nodeServerScript}\" -- --parentPid {currentProcessPid} --port {_nodeJSProcessOptions.Port}",
```

Comment:
This argument would not be necessary if one of these is true:

- Files are changed to `.mjs`
-`package.json` has `"type": "module" (requires the `package.json` to be in the `cwd()`)

This would be a better solution, so it's clear what the output is.